CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 See [RELEASE](./RELEASE.md) for workflow instructions.
 
-## UNRELEASED
+## v1.1.2
```

Comment:
i thought that we may want to put a placeholder for UNRELEASED when cutting a release, but maybe it is cleaner to add UNRELEASED section with first change. maybe will store placeholder in RELEASE.md so that it can be copied easily

Reply:
I also thought we add the UNRELEASED section back in with the first change. I'll add a template to the RELEASE.md document